projects/frontend/data-pipelines/gui/package.json

```diff
@@ -16,7 +16,8 @@
     "test": "npm run ng -- test --watch=false",
     "test:headless": "npm run test -- --browsers=ChromeHeadless_No_Sandbox",
     "lint": "npm run ng -- lint",
-    "e2e": "([ ! -z ${OAUTH2_API_TOKEN} ] || echo 'WARN: No OAUTH2_API_TOKEN set.') && npx cypress run --browser chrome --env grepTags=${FRONTEND_E2E_CYPRESS_GREP_TAGS},OAUTH2_API_TOKEN=${OAUTH2_API_TOKEN},CYPRESS_TERMINAL_LOGS='./e2e/logs' --headed",
+    "cypress:sc": "node --max_old_space_size=2048 ./node_modules/cypress/bin/cypress",
```

Comment:
Why "cypress:sc", I don't know what to make out of it? Why not simply cypress, or cypress:script (we don't have to worry about too many letters of 6 to complete the readable word)

Reply:
Also, why do we need this extracted into "scripts":{} as a separate? What is the use-case?

Reply:
I don't understand you, what you want to ask? I'm doing one script that will run cypress from local executables instead of global and provide ability to provide node env variables, in the same time I'm trying to differentiate from the cypress as alias for executable and that's why I added abbreviation ":sc" to refer it's a script and add whole world "script" because I don't find it useful.